demo/ts/components/victory-tooltip-demo.tsx

```diff
@@ -86,6 +68,35 @@ class App extends React.Component {
             ]}
           />
 
+          <VictoryCandlestick
+            horizontal
+            style={{ parent: parentStyle }}
+            highLabelComponent={<VictoryTooltip active dy={-3} />}
+            // @todo high labels type needs to have an OR so that it can also return a string.
+            // Currently, it only allows returning an integer which is incorrect, as a label would typically be a string.
```

Comment:
Can we remove this comment now you've updated the type?

Reply:
for sure